aws/sagemaker-notebook/README.md

```diff
@@ -0,0 +1,74 @@
+# Use DJL Notebook on SageMaker Notebook
```

Comment:
How will users know to find this document? It is in demodocs (while most users will probably only read the main docs), and not mentioned in any other location

Reply:
Maybe just add it to the mkdocs.yml